src/components/Book.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Books = props => {
+  const {
+    id,
+    title,
+    category,
+    handleClick,
+  } = props;
```

Comment:
Destructure props from the function parameters in functional components to avoid additional variable declarations. e.g.

```javascript
// This way uses an extra line of code and makes the component more verbose
const MyComponent = (props) => {
    const { name, age, height } = props;
    ...
}

// This way achieves the same result with less code in a more reasable way
const MyComponent = ({ name, age, height }) => {
  ...
}
```